data/pages-content/cfp.md

```diff
@@ -0,0 +1,164 @@
+us at the conference!
+
+## Topics ##
+
+We welcome presentations on any topic that will be of specific or general
+interest to members of the Python community. We are a broad and deep community,
+so talks that may, at first, seem niche or unexpected actually make EuroPython
+the cosmopolitan place that it is. **It's your conference!**
+
+Browse through our [past
+conferences](https://www.europython-society.org/europython/) and [talk
+videos](https://www.youtube.com/c/EuroPythonConference) for more ideas and to
+learn where there may be gaps that ought to be filled.
+
+For ideas to contribute to the PyData track, take a look at PyData events:
+[pydata.org](https://pydata.org/).
+
+## Selection Process ##
+
+To ensure our programme reflects our diverse community, submissions go through
+three rounds of review and refinement before the final programme is announced:
+
+* **[Community voting](/voting)** (from March 8th, until Mach 15th): is open to
+  all 2015-2024 EuroPython conference ticket holders, and anyone who proposes
+  a talk for the 2024 conference. This helps us judge both the uniqueness,
+  niche-ness, popularity and interestingness of each proposal.
+
+* **Panel feedback** (from March 8th, until Mach 15th): a diverse group of
+  experienced reviewers refine and provide constructive feedback on submissions
+  within their realm of expertise. This helps us to judge the balance, quality
+  and diversity of our programme.
+
+* **Programme team** - appointed by the EuroPython Society, these volunteers
+  curate the final programme based upon the results of community voting and
+  panel feedback. They also ensure we meet editorial criteria such as
+  diversity, depth and breadth of topics and speakers.
+
+
+## First-Time Speakers ##
+
+As in previous years, there will be a **Speaker Mentorship Programme** for
+anyone who needs help preparing their contribution. We focus on those for whom
+it might be their first time contributing to our conference, especially those
+from under-represented backgrounds. To this end we will match mentees with
+experienced and supportive mentors and run two workshops online: one close to
+the CFP closing date, focusing on improving your submissions; the other close
+to the conference date, helping you improve your talk.
+
+Mentorship Programme sign-up has started on **January 23rd** and will remain
+open until TBD February. More details can be found on our [Mentorship
+page](/mentorship).
+
+## Speaker Support
+
+We hope to meet as many speakers as possible in person. To make attendance more
+attractive, **accepted speakers are eligible for a free Conference Ticket;
+accepted workshop & tutorial organisers are eligible for a free Combined
+Ticket** (see details of the [ticket types](/tickets#ticket-types)). Travel and
+accommodation expenses will not be reimbursed,  but our [Financial Aid
+programme](/finaid) is open for applications, and we encourage speakers,
```

Comment:
This would need the finaid page to be up as well, or we can replace this with  a "More information coming soon".